README.md

```diff
@@ -1,86 +1,120 @@
 # lemur
 Bio data collector
+##TBD
```

Comment:
no reason to include this here. Use the issue tracker if you must. The README is for stuff somebody NEEDS to know in order to get going.